### PR TITLE
feat(config): Add config for customizing API server

### PR DESCRIPTION
**Pull request by arenekosreal**
This helps people who want to run their own krcparse service

### CODE REVIEW

File: package/contents/config/main.xml [hunk 1]
```diff
@@ -71,4 +75,4 @@
             <label>unhighlighted color</label>
         </entry>
     </group>
-</kcfg>
```
Comment (Sinofine) [inline package/contents/config/main.xml:74]: is this intentional?

File: package/contents/ui/configGeneral.qml [hunk 2]
```diff
@@ -1,16 +1,25 @@
-import QtQuick 2.0
+import QtQuick 2.15
```
Comment (Sinofine) [inline package/contents/ui/configGeneral.qml:1]: is this really needed?

**Review by Sinofine — COMMENTED** [review]
Have you tested the code on your machine?

File: package/contents/ui/configGeneral.qml (hunk 2 above)
Comment (arenekosreal) [reply package/contents/ui/configGeneral.qml:1]: This is needed to use RegularExpressionValidator
https://doc.qt.io/qt-5/qml-qtquick-regularexpressionvalidator.html

File: package/contents/config/main.xml (hunk 1 above)
Comment (arenekosreal) [reply package/contents/config/main.xml:74]: Removing this empty line is done by my editor and it does not result any noticeable issue, so I leave it here instead.

**Comment by arenekosreal** [comment]
> Have you tested the code on your machine?

Yes, I tested on my laptop and it works without seeing any noticeable issue

**Pull request merged**